src/NServiceBus.Core/Logging/RollingLogger.cs

```diff
@@ -175,7 +175,9 @@ void CalculateNewFileName(List<LogFile> logFiles, DateTimeOffset today)
 
         protected string currentfilePath;
         long currentFileSize;
+#pragma warning disable PS0023 // Use DateTime.UtcNow or DateTimeOffset.UtcNow - For rollover of log files, want to use local time
         internal Func<DateTimeOffset> GetDate = () => DateTimeOffset.Now.Date;
```

Comment:
Also wondering if we couldn't use UTC here, and ensure whatever is writing the value use something like `ToString("O")` to include the time zone (which it really should be doing anyway).

> 2021-07-14T07:46:23.4588246+00:00

Reply:
Don't want to change the behavior of logging in the scope of this PR.